crates/hyvectl/src/families/reqres.rs

```diff
@@ -57,8 +57,8 @@ impl CommandFamily for ReqRes {
                     };
 
                     yield output
-                    .with_tty_template("🗨  {response}")
-                    .with_non_tty_template("{response}")
+                        .with_tty_template("🗨 {response}")
```

Comment:
```suggestion
                        .with_tty_template("🗨  {response}")
```

This emoji looks ugly when there is just one whitespace